src/views/release.handlebars

```diff
@@ -1,55 +1,131 @@
 <div>
-  <div class="install-commands">
-
-    {{#ifEquals prerelease "nightly"}}
-      <pre class="command">npm install electron-nightly@{{ version }}</pre>
-      <pre class="command">yarn add electron-nightly@{{ version }}</pre>
+  <div class='install-commands'>
+    {{#ifEquals prerelease 'nightly'}}
+      <div class='command-container'>
+        <pre class='command'>npm install electron-nightly@{{version}}</pre>
+        <button class='copy-button' data-command='npm install electron-nightly@{{version}}'>
+          <i class='fas fa-copy'></i>
+        </button>
+      </div>
+      <div class='command-container'>
+        <pre class='command'>yarn add electron-nightly@{{version}}</pre>
+        <button class='copy-button' data-command='yarn add electron-nightly@{{version}}'>
+          <i class='fas fa-copy'></i>
+        </button>
+      </div>
     {{else}}
-      <pre class="command">npm install electron@{{ version }}</pre>
-      <pre class="command">yarn add electron@{{ version }}</pre>
+      <div class='command-container'>
+        <pre class='command'>npm install electron@{{version}}</pre>
+        <button class='copy-button' data-command='npm install electron@{{version}}'>
+          <i class='fas fa-copy'></i>
+        </button>
+      </div>
+      <div class='command-container'>
+        <pre class='command'>yarn add electron@{{version}}</pre>
+        <button class='copy-button' data-command='yarn add electron@{{version}}'>
```

Comment:
Add a `title="Copy to Clipboard"` attribute to these buttons so it's clear what the button does.